README.md

```diff
@@ -56,4 +83,4 @@ Example of the new values:
 }
 ```
 Result:  
-![none](https://bots.are-pretty.sexy/be34ec.png)
+![none](/be34ec.png)
```

Comment:
Until the pictures themselves are added this shouldn't be done as it breaks the result